medicai/auth/routes.py

```diff
@@ -16,14 +16,30 @@ def index():
 @cross_origin()
 def register():
     data = request.get_json()
+    if 'phone' not in data:
+        return jsonify(message="Phone number is required"), 400
+    if 'email' not in data:
+        return jsonify(message="Email is required"), 400
+    if 'password' not in data:
+        return jsonify(message="Password is required"), 400
```

Comment:
+1

Reply:
This is good validation. But I think you should validate if it's empty/blank or not. You should use regular expression to validate the format too.